datadog/api/exceptions.py

```diff
@@ -3,7 +3,20 @@
 """
 
 
-class ProxyError(Exception):
+class DatadogException(Exception):
+    """
+    Base class for Datadog API exceptions.  Use this for patterns like the following:
+
+        try:
+            # do something with the Datadog API
+        except datadog.api.exceptions.DatadogException:
+            # handle any Datadog-specific exceptions
+    """
+    def __init__(self, message, *args, **kwargs):
+        super(DatadogException, self).__init__(message, *args, **kwargs)
```

Comment:
Does not seem like `message` is useful here, I'd just remove this custom `__init__` function which does nothing.

Reply:
@jd the point of this `__init__` is to permit invocation with keyword arg `message=<bla>`, i've added some additional tests to demonstrate this.  if you feel strongly about it i'll remove it, let me know what you think

Reply:
I'm fine with it if you want message to be an argument, but since you're not using it I really don't see the value.